config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -35,8 +35,9 @@ jupyterhub:
           name: The National Institutes of Health grant 2R25MH112480-06
           url: https://reporter.nih.gov/search/ydTvTwXxk0yd6eGdRznbLQ/project-details/10409452
   singleuser:
+    # Pulls content from https://github.com/neurohackademy/nh2022-curriculum using nbgitpuller then opens up /lab
```

Comment:
```suggestion
    # Pulls content from https://github.com/neurohackademy/nh2022-curriculum using nbgitpuller then opens up /lab
    # This configuration needs to be paired with overriding schema configuration for jupyterhub-configurator, see the
    # hub.extraFiles.configurator-schema-default entry below!
```

Reply:
I edited your comment above @consideRatio. Hope that's ok. WDYT?